enclave/core/sgx/globals.c

```diff
@@ -196,6 +198,39 @@ size_t __oe_get_reloc_size()
 #endif
 }
 
+/*
+**==============================================================================
+**
+** blob boundaries:
+**
+**==============================================================================
+*/
+
+const void* __oe_get_blob_base()
```

Comment:
Are these intended to be public functions called by enclave developers (it seems no because its in an internal header)? If enclave writers are expected to be able to use this feature it probably makes sense to provide a public API and a reasoning for why one would use it.